test/unit/containerDeploymentWebviewController.test.ts

```diff
@@ -406,13 +413,13 @@ suite("ContainerDeploymentWebviewController", () => {
                 DockerStepOrder.connectToContainer
             ].errorMessage.includes("dev-profile"),
         );
-        assert.ok(sendErrorEvent.calledTwice, "sendErrorEvent should be called twice");
+        assert.ok(sendErrorEvent.calledOnce, "sendErrorEvent should be called twice");
 
         addContainerConnectionStub.restore();

```

Comment:
restore call isn't necessary since the stub is created in the same test.  this is for when you need to "unstub" something for later execution, but in this case, it's all cleaned up.